facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java

```diff
@@ -603,49 +597,82 @@ private org.jboss.pnc.model.Artifact.Builder mapArtifact(Artifact artifact, User
         return builder;
     }
 
-    private static String createDeployPath(MavenArtifact mavenArt) {
-        String filename = createFileName(mavenArt);
-        String deployPath = "/" + mavenArt.getGroupId().replace('.', '/') + "/" + mavenArt.getArtifactId() + "/"
-                + mavenArt.getVersion() + "/" + filename;
+    private static String createDeployPath(Artifact artifact) {
+        String filename = createFileName(artifact);
+        String deployPath;
+
+        if (artifact instanceof MavenArtifact) {
+            MavenArtifact mavenArtifact = (MavenArtifact) artifact;
+            deployPath = "/" + mavenArtifact.getGroupId().replace('.', '/') + "/" + mavenArtifact.getArtifactId() + "/"
+                    + mavenArtifact.getVersion() + "/" + filename;
+        } else if (artifact instanceof WindowsArtifact) {
+            deployPath = "/" + filename;
```

Comment:
@pkocandrle I am not sure how the deploy path would be if we got that artifact via brew pull (is it even possible?) and if this matches.

Reply:
As I understand the code, this isn't the Maven repository path, e.g., /maven2. This is the original Brew path (see `createBrewOriginURL()`).

Maven artifacts start with /maven and Windows artifacts start with /win, so I think it just uses the artifact type to start the path. For Windows, since there is no group identifier, etc., the rest of the path is just the filename itself.